### ISSUE TITLE
Failing build - composer is not available on your system

### ISSUE THREAD

**Issue by HanaGemela** [owncloud/files_primary_s3#328]
When running the docker build script, the below error occurs

```
make[1]: Entering directory '/var/www/owncloud/repo/files_primary_s3'
Makefile:143: *** composer is not available on your system, please install composer.  Stop.
make[1]: Leaving directory '/var/www/owncloud/repo/files_primary_s3'
make: *** [Makefile:57: dist] Error 2
```

**Comment by phil-davis** [comment]
I guess that you have a computer that does not have composer installed.
That is rather unusual (for developers :)

I will make a PR  to adjust that - it has been done already in other repos. Probably we need to do it in all repos. (Or you could install composer)

**Comment by HanaGemela** [comment]
Isn't it all done in the docker image?
And the script used to work with files primary s3 1.1.1

**Comment by phil-davis** [comment]
@HanaGemela I guess I am not understanding.
You are typing `make dist` on your local computer?
Or is this something that happens "in the cloud" in drone?

**Comment by HanaGemela** [comment]
All the apps are build in a docker container with the build script (same script as mentioned https://github.com/owncloud/files_lifecycle/issues/202#issuecomment-601593784)

**Comment by phil-davis** [comment]
OK - I can get an error locally, even when I have composer.
It is related to PR https://github.com/owncloud/files_primary_s3/pull/325/files

Looking...

**Comment by phil-davis** [comment]
Should be fixed by #329

**Comment by phil-davis** [comment]
And #330 fixes it in the release branch.
